test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -112,6 +112,22 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
         }
 
         [Fact]
+        public async Task ViewReturnsPage()
```

Comment:
[This](https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs#L69) test covers `void OnGet` by hitting [here](https://github.com/aspnet/Mvc/blob/dev/test/WebSites/RazorPagesWebSite/HelloWorldWithHandler.cshtml#L6).